internal/db/models/project.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Project struct {
+	gorm.Model
+	ID          uint      `json:"-" gorm:"primaryKey"`
+	OwnerID     uint      `json:"-" gorm:"not null; index"`
+	Name        string    `json:"name" gorm:"not null; index"`
+	Description string    `json:"description" gorm:"type:text"`
+	Config      string    `json:"config" gorm:"type:text"`
+	Tasks       []Task    `json:"tasks" gorm:"foreignKey:ProjectID"`
```

Comment:
I think this should just be  the task IDs. so `TaskIDs []uint`

Reply:
Or does gorm automatically handle that under the hood? 

Basically I don't think we should be storing the full task info in the projects DB, just the reference to the task in the tasks db